packages/extension/src/inpage/index.ts

```diff
@@ -68,58 +68,56 @@ window.addEventListener("load", () => attachHandler())
 document.addEventListener("DOMContentLoaded", () => attachHandler())
 document.addEventListener("readystatechange", () => attachHandler())
 
+// TODO: check if this listener is necessary
```

Comment:
This is still needed probably, when users change accounts or switch networks on extension, dApp gets the chance to react automatically here.

Reply:
I think dApp does not need to change. Image the scenarios that multiple dApps use different addresses in parallel:
1. Nostr client uses a Schnorr address for signing messages
2. One dApp uses a hot wallet for small transactions
3. One dApp uses a hardware wallet for more important stuff
4. dApps uses addresses from multiple groups

In extension accounot switching should not affect those dApps, otherwise it's not user-friendly. WDYT?